src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -106,7 +106,6 @@ class AltDiffusionPipeline(DiffusionPipeline, TextualInversionLoaderMixin, LoraL
         feature_extractor ([`~transformers.CLIPImageProcessor`]):
             A `CLIPImageProcessor` to extract features from generated images; used as inputs to the `safety_checker`.
     """
-
```

Comment:
Lets keep the blank line here, it is causing the test to fail (same for the `pipeline_alt_diffusion_img2img.py` file).

Reply:
I believe the change is automatically generated by the "make fix-copies" command. I will investigate why it's being added and remove the commit that made this change.

Reply:
Ah yeah, I had a similar issue and I resolved it by removing the blank lines and not running the `make fix-copies` command.